Makefile

```diff
@@ -1,8 +1,6 @@
 .PHONY: help
 help:
 	@echo "make help              Show this help message"
-	@echo 'make services          Run the services that `make dev` requires'
-	@echo "                       (Postgres) in Docker"
```

Comment:
`make services` still exists but is now undocumented because you never need to run it manually